src/hostname/hostnamectl.c

```diff
@@ -107,6 +109,11 @@ static void print_status_info(StatusInfo *i) {
         if (!isempty(i->architecture))
                 printf("      Architecture: %s\n", i->architecture);
 
+        if (!isempty(i->hardware_vendor))
+                printf("    HardwareVendor: %s\n", i->hardware_vendor);
```

Comment:
please add space between "Hardware" and "Vendor", this should be English language